### ISSUE TITLE
Hardcoded Icons

### ISSUE THREAD

**Issue by Foggalong** [linuxmint/mint-x-icons#43]
I'll start off by apologising for my unorthodox method of getting this message accross I've tried a few email address but I think I got the wrong one so I thought I'd try at the source instead.

I'm trying to develop a standardised solution to the problem of hardcoded icons. A link to the project with an explanation of what it does can be found [here](https://github.com/Foggalong/hardcode-fixer) but I'll try to explain it briefly here. I'm looking for icon themes to get involved with it and currently elementary XFCE, Numix, Moka, KotusWorks, and Nitrux are on board.

At the minute to correct a hardcoded icon the end user has to manually change launcher files to icon theme specific variables. With this script the icons are safely and easily switched to present names which avoid breakage even when the theme is changed (as currently happens).

It does this by changing the icon name of the launcher to a intuitive icon name and then copying the original icon to the hicolor directory with the new intuitive name. Of course this just  a workaround solution and so I'm also putting a lot of effort into reporting upstream reports.

For the icon themes all it means is when/if they create icons for any of the applications they create a one (or a symlink) to one with the standardised name. Hope this sounds like something you might want to get involved with, and I look forward to hearing back from you.

**TL;DR: I'm making a safe solution to hardcoded icons and want icon themes on board - does Mint-X want to be part of it?**
